resources/v2/03_bookings.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE bookings (
+    id INTEGER NOT NULL PRIMARY KEY,
+    pickup TEXT NOT NULL,
+    destination TEXT NOT NULL,
+    user_id INTEGER NOT Null,
+    cab_id INTEGER NOT Null,
+    created_at timestamp WITH time zone DEFAULT NOW(),
+    updated_at timestamp WITH time zone,
+    deleted_at timestamp WITH time zone,
```

Comment:
Make it `canceled_at` instead of `deleted_at`

Reply:
done